next.config.js

```diff
@@ -0,0 +1,4 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {}
```

Comment:
```suggestion
const nextConfig = {
  experimental: { esmExternals: 'loose' },
};
```
For fixing error: Module not found: ESM packages (d3-shape) need to be imported.
See the link: https://nextjs.org/docs/messages/import-esm-externals @justin-kleid

Reply:
@justin-kleid Did the above change fix the error "EMS package(d3-shape) need to be imported"?